### ISSUE TITLE
Moving between tech jobs

### ISSUE THREAD

**Issue by nitishdayal** [shanselman/ama#25]
Thanks for taking the time to answer questions.

tldr; Other industries frown upon frequently moving between companies and/or positions, even if that movement is upwards. Is tech the same way? Is there a reasonable 'wait time'?  Would you recommend against leaving a stable position at an enterprise company to explore what else might be out there? 

I was fortunate enough to find my way into a development position with Amazon a few months back. After I was hired, I've been contacted about positions with a few other companies. I enjoy my job and the team I work with at Amazon and have little incentive to leave...that I know of. Who knows, maybe one of these companies that contacted me will be working on something magical that I 100% want to be a part of. The opportunities seem worth exploring. Would it reflect poorly on me for moving between companies in a relatively short time frame? Or is the beginning of a career in tech the right time to be exploring? Any insight/opinions would be appreciated.

**Comment by shanselman** [comment]
This is my opinion, but I really think that spending a few (1-2) years at a company to start is important. If it is a healthy environment and you are learning, then stick with it. Jumping around with 3 months here and 8 months there on a resume does raise some flags. Again, assuming a healthy and fulfilling environment.
